GAMMA/RTC/README_RTC_GAMMA.txt

```diff
@@ -1,11 +1,16 @@
 ASF RTC Data Package (GAMMA)
 ============================
 
-This folder contains radiometric terrain corrected (RTC) products and their associated files. They were generated by the Alaska Satellite Facility (ASF) HYP3 RTC software version [HYP3_VER], running GAMMA release [GAMMA_VER]. They are projected to [PCS], and the pixel spacing is [SPACING] m.
+<!-- Consider opening this document in a Markdown editor/viewer for easier reading -->
```

Comment:
I would prefer that this be a little further down so it doesn't distract from the initial reading of the information. Perhaps right after the granule name (line 33).

Reply:
I think personally think it it should be upfront so that readers who are interested/knowledgeable can switch viewers immediately. But, I can see having it out of the content area, so I might actually prefer it as the first line like:

```md
<!-- Consider opening this document in a Markdown editor/viewer for easier reading -->

ASF RTC Data Package (GAMMA)
============================
```

which will render in most markdown viewers like: 

---
<!-- Consider opening this document in a Markdown editor/viewer for easier reading -->

ASF RTC Data Package (GAMMA)
============================

---

What does everyone else think?